WooCommerce/Classes/Extensions/WordPressAuthenticator+Woo.swift

```diff
@@ -8,7 +8,7 @@ extension WordPressAuthenticator {
                                             featureFlagService: FeatureFlagService = ServiceLocator.featureFlagService) {
         let isWPComMagicLinkShownAsSecondaryActionOnPasswordScreen = true
         let isManualErrorHandlingEnabled = featureFlagService.isFeatureFlagEnabled(.manualErrorHandlingForSiteCredentialLogin)
-        let noMoreWooExpressSignup = featureFlagService.isFeatureFlagEnabled(.noMoreWooExpressSignup)
+        let noMoreWooExpressSignup = true
```

Comment:
Nit: you can remove this declaration and set the appropriate values to the config below for easier readability.

Reply:
Oh, I did it this way thinking it will be easier to modify later in one place if needed. But your point makes sense. updated in https://github.com/woocommerce/woocommerce-ios/pull/12414/commits/e7c43c1368de1a5f520293da43714e6ef62e34ec